GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -95,12 +96,18 @@ public sealed partial class FormBrowse : GitModuleForm, IBrowseRepo
         [CanBeNull] private readonly IAheadBehindDataProvider _aheadBehindDataProvider;
         private readonly WindowsJumpListManager _windowsJumpListManager;
         private readonly ISubmoduleStatusProvider _submoduleStatusProvider;
+        [CanBeNull] private SubmoduleInfoResult _submoduleInfoResult;
+        private readonly IWorktreeStatusProvider _worktreeStatusProvider;
+        [CanBeNull] private WorktreeInfoResult _worktreeInfoResult;
         private readonly FormBrowseDiagnosticsReporter _formBrowseDiagnosticsReporter;
         [CanBeNull] private BuildReportTabPageExtension _buildReportTabPageExtension;
         private readonly ShellProvider _shellProvider = new();
         private ConEmuControl _terminal;
         private Dashboard _dashboard;
 
+        // Do not update combo box twice or more if several git commands are executed (submodules, worktrees)
+        private bool _notUpdateWorkingDirComboText;
```

Comment:
Positive logic is easier to read, isn't it?
```suggestion
        private bool _workingDirComboTextsLoaded;
```